api/lib/infrastructure/repositories/knowledge-element-repository.js

```diff
@@ -42,7 +42,7 @@ async function _findByCampaignIdForSharedCampaignParticipationWhere(campaignPart
 
 function _getByUserIdAndLimitDateQuery({ userId, limitDate }) {
   return Bookshelf.knex
-    .select('*', Bookshelf.knex.raw('ROW_NUMBER() OVER (PARTITION BY ?? ORDER BY ?? DESC) AS rank', ['skillId', 'createdAt']))
+    .select('*')
     .from('knowledge-elements')
     .where((qb) => {
```

Comment:
```suggestion
    .where((ke) => {
```
j'ai mis KE pour knowledge element
qb  c'est pour Query Builder ?

Reply:
oui oui c'est query builder !